packages/uni_app/lib/http/utils.dart

```diff
@@ -0,0 +1,19 @@
+import 'dart:io';
+
+import 'package:http/http.dart' as http;
+
+/// Extracts the cookies present in [response].
+List<Cookie> extractCookies(http.Response response) {
+  final setCookieHeaders =
+      response.headersSplitValues[HttpHeaders.setCookieHeader];
+  if (setCookieHeaders == null) {
+    return [];
+  }
+
+  final cookies = <Cookie>[];
+  for (final value in setCookieHeaders) {
+    cookies.add(Cookie.fromSetCookieValue(value));
+  }
+
+  return cookies;
```

Comment:
Probably it's best to use `.map` here

```suggestion
  final cookies = setCookieHeaders.map(Cookie.fromSetCookieValue).toList();
```

Reply:
Yeah, thanks!